src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -441,6 +492,68 @@ const EditHomepage = ({ match }) => {
           scrollTo(scrollRefs[0])
           break
         }
+        case "announcement": {
+          // The field that changed belongs to a announcements
```

Comment:
```suggestion
          // The field that changed belongs to announcements
```

Reply:
differing meaning, modified to this instead for clarity ce2460f28f5f85fe4c380b5d1c15b16b184313b0